precog/miners/base_miner.py

```diff
@@ -23,8 +23,8 @@ def get_point_estimate(timestamp: str) -> float:
 
     # Set the time range to be as small as possible for query speed
     # Set the start time as 2 seconds prior to the provided time
-    start_time: str = datetime_to_CM_timestamp(iso8601_to_datetime(timestamp) - timedelta(days=1))
-    end_time: str = datetime_to_CM_timestamp(iso8601_to_datetime(timestamp))  # built-ins handle CM API's formatting
+    start_time: str = datetime_to_iso8601(iso8601_to_datetime(timestamp) - timedelta(seconds=2))
```

Comment:
We discussed passing actual datetime object in calculations to help clean up lines like this.  Would we want to do in a later PR, to only change one thing at a time, or batch the change here?

Reply:
let's batch the change in this pr. i'll play with alternative ways to write this line. i suspect it only appears complex because the function names are long